app/src/main/java/com/github/mik629/android/fundamentals/di/AppModule.kt

```diff
@@ -45,4 +62,39 @@ class AppModule {
             return chain.proceed(newRequest)
         }
     }
+
+    fun provideMovieListViewModel(context: Context): MoviesListViewModel =
```

Comment:
remove vm?

Reply:
why? it's being provided by factory but via AppModule

Reply:
Maybe I should already introduce dagger?

Reply:
Done, added dagger